lib/challenge.js

```diff
@@ -19,6 +19,36 @@
   // - park
 
 // Answer Starts Here
+// Year
+// Description: Represents one year
+// Every year has
+// - Identifier  IE 2015
+// - 12 months
+// Every Year can
+// - Diplay or Print itself show days with appointments
+// - return a specific month
+
+// 'Month'
+//  Description:  represents a month in the year
+//  Every 'month' has
+// - a name
+// - a array of days
+// Every Month Can
+// - Display or Print itself showing days with appointments
+// - return a specific day
+
+// 'Day'
+// Description:  represents a single day
+// Every Day has
+// - a name = day of week
+// - a list of appointments
+//
+//  Every day can
+// - Set a new appointment
+// - Delete an existing appointment
+
+
```

Comment:
Sure, not unreasonable! Some other abstractions we might consider for an application like this might be, for instance, 'Users', 'Events', or 'Calendars' (as in, individual calendars within the app).